File: ShipperManagerWeb/ShipperManager/Models/Shipper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ShipperManager.Models
{
    public class Shipper
    {
        public string Id { get; set; }
        public string Ten { get; set; }
        public string QueQuan { get; set; }
        public string TaiKhoan { get; set; }
        public string MatKhau { get; set; }
        public string NgaySinh { get; set; }
        public bool TrangThai { get; set; }
    }
}

File: ShipperManagerAndroid/app/src/main/java/com/example/shippermanager/Model/Shipper.java
package com.example.shippermanager.Model;

public class Shipper {

    public String Id;
    public String Ten;
    public String NgaySinh;
    public String QueQuan;
    public String TaiKhoan;
    public String MatKhau;
    public boolean TrangThai;
    public double KinhDo;
    public double ViDo;
    public String ImagePath;

    public Shipper() {

    }

    public Shipper(String id, String ten, String ngaySinh, String queQuan, String taiKhoan, String matKhau, boolean trangThai, double kinhDo, double viDo,String imagePath) {
        Id = id;
        Ten = ten;
        NgaySinh = ngaySinh;
        QueQuan = queQuan;
        TaiKhoan = taiKhoan;
        MatKhau = matKhau;
        TrangThai = trangThai;
        KinhDo = kinhDo;
        ViDo = viDo;
        ImagePath = imagePath;
    }
}
